x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -162,12 +183,16 @@ export class CspPlugin
 
   public stop() {}
 
+  /**
+   * Initialization is idempotent and required for (re)creating indices and transforms.
+   */
   async initialize(core: CoreStart, taskManager: TaskManagerStartContract): Promise<void> {
-    this.logger.debug('initialize');
+    this.#initializerTracker.off();
```

Comment:
I don't think `off` describes the actual step correctly - it is being initialized at the moment. But it could be already initialized - which is ok.

In other words, once initialized - always initialized.

worth only having a log message instead - to trace initialize is being called.